fineract-provider/src/main/resources/sql/migrations/core_db/V379__user_cannot_change_password.sql

```diff
@@ -0,0 +1,20 @@
+--
```

Comment:
I've verified that the last one in https://github.com/apache/fineract/tree/develop/fineract-provider/src/main/resources/sql/migrations/core_db is 378, so this at 379 LGTM.